setup.py

```diff
@@ -1,20 +1,21 @@
 from setuptools import setup
 
 setup(name='ViralFlow',
-      version='0.0.6.dev',
+      version='1.0.0',
```

Comment:
Tonho, vamos jogar na versão 1.0.0 já? De acordo com o versionamento semantico, a versão 1.0.0 é a primeira versão pronta pra produção, nessa linha, estamos considerando que o viralflow ta pronto, e que tudo que vier agora será melhoria.

Se ja tiver tudo ok, pode manter na `1.0.0`, se não, recomendo por `0.1.0` e aí vamos aumentando de acordo com as próximas correções, ou `1.0.0-alpha` se for considerar o readme do vfnext

Reply:
Done